src/ROUTES.ts

```diff
@@ -185,6 +185,10 @@ const ROUTES = {
         route: 'r/:reportID/avatar',
         getRoute: (reportID: string) => `r/${reportID}/avatar` as const,
     },
+    REPORT_WITH_ID_AND_ACTION_ID: {
```

Comment:
Why a separate route for this? Can we include it in `REPORT_WITH_ID`, which already has something for `:reportActionID`?

Reply:
Yeah, it makes sense to combine it, I'll do it in a moment